### ISSUE TITLE
research/object_detection/dataset_tools/create_pet_tf_record.py obviously mixes up the file names

### ISSUE THREAD

**Issue by rudevel** [tensorflow/models#4006]
Please **READ BEFORE** you go and watch the error- message- 
As the **error- message** is **totally misleading**:

**_TL;DR;: Swap the if-else branches 285-288_**

In **lines 285 - 288** the scripts applies names _"_with_masks_"_ if **FLAGS.faces_only**.

But according to the description (AND my mask training _finally_ working) this is exactly the switch that causes NO masks to be generated. 

Additional flaws: 
- With PNG the file sizes are NOT significantly larger as the parameter description says (another confusion point)
- Folders do not work as the parameters suggest (only pwd)- but that is (partly) covered by the pets- howto (and symlinks to images and annotations did work)
- With this minor problem I get misleading error messages like the following:

...
assertion failed: [] [Condition x == y did not hold element-wise:] [x (Loss/BoxClassifierLoss/assert_equal_2/x:0) = ] [0] [y (Loss/BoxClassifierLoss/assert_equal_2/y:0) = ]
...

While this is for sure a result of the wrong input it has most probably already - and will for sure - cause many others to be confused and search for hours.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
